gnomad/assessment/validity_checks.py

```diff
@@ -1232,6 +1233,11 @@ def check_missingness_of_struct(
             )
             for key in struct_expr.keys()
         }
+    elif isinstance(struct_expr, hl.expr.ArrayExpression):
+        # Count array as missing if it is NA, an empty array, or only has missing elements.
+        return hl.agg.fraction(
+            hl.or_else(struct_expr.all(lambda x: hl.is_missing(x)), True)
```

Comment:
i get a different answer if I change it, as it would skip counting the example where d is just completely missing (hl.struct(a=3, b='bloop', c=hl.struct(d=hl.missing(hl.tarray(hl.tstr)), e='yikes')